### PR TITLE
feat: add error comment in lint-pr-title.yml

### PR DESCRIPTION
**Pull request by Krishks369**
I have added feature in the lint-pr-title.yml file to comment on the lint errors in the title while PR is created

 **Three scenarios are there for the workflow**

1.  When the PR title is not in the correct format i.e when it is apart from titles like `feat`, `fix` etc. It works like it has in this [PR](https://github.com/Krishks369/.github/pull/16) .

2. When the PR title is not found or the PR subject starts in capital, then it works like in this [PR](https://github.com/Krishks369/.github/pull/17)

3. When the lint is corrected after the comment, the comment is then deleted from the PR, as mentioned in https://github.com/marocchino/sticky-pull-request-comment

Although the action used for linting the title is https://github.com/amannn/action-semantic-pull-request  I have forked and used this feature before it got merged in the main action. So temporarily it uses https://github.com/Krishks369/action-semantic-pull-request.

### CODE REVIEW

File: .github/workflows/lint-pr-title.yml [hunk 1]
```diff
@@ -1,23 +1,46 @@
...
+          message: |
+                   ```
+                    ${{ steps.lint_pr_title.outputs.error_message}}
+                   ```
+       # deletes the error comment if the title is correct            
+      - if: ${{ steps.lint_pr_title.outputs.error_message == null }}
+        name: delete the comment
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:   
+          header: comment
```
Comment (derberg) [inline .github/workflows/lint-pr-title.yml:43]: you tested it, right?

**Review by derberg — CHANGES_REQUESTED** [review]
I left some comments

**Comment by derberg** [comment]
@Krishks369 hey man, need a hand?

**Comment by sambhavgupta0705** [comment]
Hey @derberg ,do we need to refine this PR also?

**Pull request closed without merging**